static/bidder-params/generic.json

```diff
@@ -0,0 +1,13 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Generic Adapter Params",
+    "description": "A schema which validates params accepted by the Generic adapter",
+    "type": "object",
+    "properties": {
+      "exampleProperty": {
+        "type": "string",
+        "description": "Example property",
+        "minLength": 1
+      }
+    }
+  }
```

Comment:
It's invalid in PBS-Go for aliases to specify a different json schema. Does PBS-Java allow generic aliases to specify their own schema?